erpnext/agriculture/doctype/strain/strain.py

```diff
@@ -32,6 +37,52 @@ def validate_strain_tasks(self):
 		# Sort the strain tasks based on start days,
 		# maintaining the order for same-day tasks
 		self.cultivation_task.sort(key=lambda task: task.start_day)
+	
+	def update_to_bloomtrace(self):
+		"""
+		Create a new strain on Bloomtrace via Bloomstack.
+		Args: 
+		
+		Returns: 
+			Successful Message to the User
+		"""
+		now = datetime.now()
+		try:
+			data = self.as_dict()
+			data['owner'] = "neil@bloomstack.com"
+			data['modified_by'] = "neil@bloomstack.com"
+			request_data = {
+				"site_url": "manufacturing.bloomstack.io",
+				"customer_name": "Bloomstack",
+				"company_name": "Bloomstack India",
+				"license_number": "A12-0000015-LIC",
+				"Hello": "World",
+				"rest_method": "POST",
+				"environment": "sandbox",
+				"rest_operation": "Strains Create",
+				"doctype": "Strain",
+				"doctype_data": data
+			}
+			# create a strain on the bloomtrace
+			bloomtrace_response = requests.post('https://bl2qu9obqb.execute-api.ap-south-1.amazonaws.com/dev/doctype/createstrain', json=request_data)
```

Comment:
the url has to come from bloomtrace settings doctype

Reply:
yes, creating a PR for that